api/tag.go

```diff
@@ -98,6 +98,8 @@ func (h TagHandler) Create(ctx *gin.Context) {
 		return
 	}
 	m := r.Model()
+	// Load associated TagCategory.
+	h.DB(ctx).First(&m.Category, m.CategoryID)
```

Comment:
There are likely many places where resources are created with refs.  Further we could (and probably do) create resources with nested resources containing refs.  This is likely a _can of worms_ to be avoid.
For now, I wonder if we can **not** include this change and set the expectation that POST operations will only return resources with nameless refs?

Reply:
Agreed on can of worms setting Name on all such places. Keeping Ref with ID mandatory (as until now), but setting right expectation, that the Name really don't have to be provided (so will be omitted) sounds good to me too!